rdfs-improved/fix01-whitespace-6.ru

```diff
@@ -0,0 +1,13 @@
+﻿# https://github.com/Sveino/Inst4CIM-KG/tree/develop/rdfs-improved#whitespace-in-definitions
```

Comment:
@Sveino @griddigit-ci also read this (SPARQL update). Does it use any tricks you don't know?
Do you understand what it does?
The very first line suggests what it does (it's a link to the README section)